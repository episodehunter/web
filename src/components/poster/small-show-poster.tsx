import * as React from 'react'
import { images } from '../../images.config'
import { Poster } from './poster'

type Props = {
  tvdbId: number
}

export const SmallShowPoster = ({ tvdbId }: Props) => (
  <Poster width={185} height={273} imagePath={images.poster.small(tvdbId)} />
)
