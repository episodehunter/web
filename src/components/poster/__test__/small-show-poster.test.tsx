import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { SmallShowPoster } from '../small-show-poster'

it('SmallShowPoster renders correctly', () => {
  const tree = renderer.create(<SmallShowPoster tvdbId={123} />).toJSON()
  expect(tree).toMatchSnapshot()
})
