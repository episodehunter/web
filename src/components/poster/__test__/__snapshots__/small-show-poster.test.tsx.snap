// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallShowPoster renders correctly 1`] = `
<img
  defaultImage="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="
  height={273}
  image="https://d1lolx4ilifvdr.cloudfront.net/poster/185x273/123.jpg"
  style={
    Object {
      "flexShrink": 0,
    }
  }
  width={185}
/>
`;
